upt_macports/templates/python.Portfile

```diff
@@ -3,7 +3,34 @@
 {% block portgroup %}
 PortGroup           python 1.0
 {% endblock %}
+{% block nameversion %}
+name                {{ pkg._pkgname() }}
+version             {{ pkg.upt_pkg.version }}
+revision            0
+{% endblock %}
+
+{% block dist_info %}
+homepage            {{ pkg.upt_pkg.homepage }}
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+{% endblock %}
 
 {% block versions %}
-python.versions     27 36 37
+python.versions     37
```

Comment:
We need to find a solution for this that requires some more care, but proper implementation is likely too much for this very moment. If a package is not compatible with python 3, we need to add just `27`. If it's not compatible with python 2, we need to add just `37`. If it's compatible with both, it would probably be both `27 37`.

Reply:
Okay